docs/install.md

```diff
@@ -12,50 +12,63 @@ weight=4
+      
+3. Apply executable permissions to the binary:
 
-    curl -L https://github.com/docker/compose/releases/download/1.3.3/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
-    chmod +x /usr/local/bin/docker-compose
+        $ chmod +x /usr/local/bin/docker-compose
 
-> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory probably isn't writable and you'll need to install Compose as the superuser. Run `sudo -i`, then the two commands above, then `exit`.
+4.  Optionally, install [command completion](completion.md) for the
+`bash` and `zsh` shell.
 
-Optionally, you can also install [command completion](completion.md) for the
-bash and zsh shell.
+5. Test the installation.
 
-> **Note:** Some older Mac OS X CPU architectures are incompatible with the binary. If you receive an "Illegal instruction: 4" error after installing, you should install using the `pip` command instead.
+        $ docker-compose --version
+        docker-compose version: 1.4.0rc3
 
-Compose is available for OS X and 64-bit Linux. If you're on another platform,
-Compose can also be installed as a Python package:
+## Upgrading
 
-    $ sudo pip install -U docker-compose
+If you're upgrading from Compose 1.2 or earlier, you'll need to remove or migrate
+your existing containers after upgrading Compose. This is because, as of version
+1.3, Compose uses Docker labels to keep track of containers, and so they need to
+be recreated with labels added.
 
-No further steps are required; Compose should now be successfully installed.
-You can test the installation by running `docker-compose --version`.
+If Compose detects containers that were created without labels, it will refuse
+to run so that you don't end up with two sets of them. If you want to keep using
+your existing containers (for example, because they have data volumes you want
+to preserve) you can migrate them with the following command:
 
-### Upgrading
+    $ docker-compose migrate-to-labels
 
-If you're coming from Compose 1.2 or earlier, you'll need to remove or migrate your existing containers after upgrading Compose. This is because, as of version 1.3, Compose uses Docker labels to keep track of containers, and so they need to be recreated with labels added.
+Alternatively, if you're not worried about keeping them, you can remove them &endash;
+Compose will just create new ones.
 
-If Compose detects containers that were created without labels, it will refuse to run so that you don't end up with two sets of them. If you want to keep using your existing containers (for example, because they have data volumes you want to preserve) you can migrate them with the following command:
-
-    docker-compose migrate-to-labels
-
-Alternatively, if you're not worried about keeping them, you can remove them - Compose will just create new ones.
-
-    docker rm -f myapp_web_1 myapp_db_1 ...
+    $ docker rm -f myapp_web_1 myapp_db_1 ...
```

Comment:
I think it's better to use `docker rm -v -f myapp_ ....` (add the `-v` flag), so that volumes are also removed, and the user doesn't end up with orphaned volumes